yt/data_objects/static_output.py

```diff
@@ -1717,7 +1728,7 @@ def quan(self):
         return self._quan
 
     def add_field(
-        self, name, function, sampling_type, *, force_override=False, **kwargs
+        self, name, function, *, sampling_type=None, force_override=False, **kwargs
```

Comment:
So this is a slightly significant change in behavior, isn't it? We've previously mandated that the `sampling_type` be specified. Any concerns about this? I don't have any, but I wondering if anyone else has any ideas about potential downstream effects.